packages/checkout-com/src/useCko.ts

```diff
@@ -103,16 +102,11 @@ const useCko = () => {
     if ([CkoPaymentType.CREDIT_CARD, CkoPaymentType.SAVED_CARD].includes(selectedPaymentMethod.value)) {
       finalizeTransactionFunction = makeCardPayment;
       localError = cardError;
-    } else if (selectedPaymentMethod.value === CkoPaymentType.KLARNA) {
-      finalizeTransactionFunction = () => {
-        console.log('Making transaction with Klarna...');
-      };
     } else if (selectedPaymentMethod.value === CkoPaymentType.PAYPAL) {
-      finalizeTransactionFunction = () => {
-        console.log('Making transaction with PayPal...');
-      };
+      finalizeTransactionFunction = makePaypalPayment;
+      localError = paypalError;
     } else {
-      error.value = 'Not supported payment method';
+      error.value = new Error('Not supported payment method');
```

Comment:
why you are returning a new error instead of string?

Reply:
When we have `try{...}catch(e){error.value=e;}` parts - `e` is instance of `Error`. I wanted to unify errors in whole package